integration-tests/eventbridge/nino-check/nino-check-publish.test.ts

```diff
@@ -0,0 +1,285 @@
+import { stackOutputs } from "../../resources/cloudformation-helper";
+import { executeStepFunction } from "../../resources/stepfunction-helper";
+import {
+  clearItemsFromTables,
+  clearAttemptsTable,
+  populateTables,
+} from "../../resources/dynamodb-helper";
+import { CreateQueueCommandOutput } from "@aws-sdk/client-sqs";
+import {
+  deleteQueue,
+  getQueueMessages,
+  setUpQueueAndAttachToRule,
+  targetId,
+} from "../../resources/queue-helper";
+import { removeTargetFromRule } from "../../resources/event-bridge-helper";
+import { pause, retry } from "../../resources/util";
+
+jest.setTimeout(650_000);
```

Comment:
This looks significantly longer than most other timeouts - is there a reason for that?

Reply:
This is testing an async system so throughout the tests you'll see some bits such as `await pause(60);` - the global timeout is to cater for all the additional time the test is going to take otherwise the test will keep timing out.